app/views/start-page.html

```diff
@@ -1,31 +1,66 @@
 
 {% extends "layout.html" %}
+{% set bodyClasses = "start-page" %}
 
-{% block content %}
+{% block main %}
+  <div class="govuk-width-container">
+    {{ govukPhaseBanner({
+      tag: {
+        text: "prototype"
+      },
+      html: 'This is a prototype of a new service. Some parts of this prototype do not work yet. '
+    }) }}
+    {# this overrides the default phase banner from layout.html #}
+    {% block beforeContent %}
+    {% endblock %}
+  </div>
+  <div class="start-page-banner">
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <h2 class="govuk-heading-l">Create a trainee record</h2>
-        <p class="govuk-body">Complete the training record, check it and send for TRN registration.</p>
-        <p class="govuk-body">The training record is divided into:</p>
-
-        <ul class="govuk-list govuk-list--bullet">
-          <li>personal details</li>
-          <li>contact details</li>
-          <li>previous education</li>
-          <li>training details</li>
-          <li>course details</li>
-        </ul>
-
-        {{ govukButton({
-          text: "Start record",
-          href: "/personal-details.html",
-          isStartButton: true
-        }) }}
+    <div class="govuk-width-container">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+          <h1 class="start-page-banner__text start-page-banner__heading govuk-heading-xl">Register trainee teachers</h1>
+          <p class="start-page-banner__text start-page-banner__description govuk-body govuk-!-font-size-27">A service for collecting and maintaining trainee teacher registration data.</p>
+          <div class="start-page-banner__actions">
+            {{ govukButton({
+              text: "Get started",
+              href: "/records.html",
```

Comment:
```suggestion
              href: "/records",
```

We don't need to provide an extension here. I've now removed similar from other buttons.

Reply:
Good shout